source/common/util/ignore-dir.ts

```diff
@@ -16,20 +16,29 @@ import path from 'path'
 // Ignored directory patterns
 import { ignoreDirs } from '../data.json'
```

Comment:
I believe pulling in those regexps here into this file is a good idea, since they aren't referenced anywhere else. (Also this data file is a remnant from way back, where nothing was consolidated)